OracleGoldenGate/setupLinuxEnv.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# LICENSE CDDL 1.0 + GPL 2.0
+#
+# Copyright (c) 1982-2016 Oracle and/or its affiliates. All rights reserved.
+#
+# Since: December, 2016
+# Author: gerald.venzl@oracle.com
+# Description: Sets up the unix environment for DB installation.
+# 
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+# 
+# Setup filesystem and oracle user
+# Adjust file permissions, go to /opt/oracle as user 'oracle' to proceed with Oracle installation
+# ------------------------------------------------------------
+
+# Setup environment for GoldenGate.   
+mkdir -p $OGG_INSTALL_DIR && \
+chown -R oracle:oinstall $INSTALL_DIR && \
+chown -R oracle:oinstall $OGG_INSTALL_DIR && \
+chmod ug+x $INSTALL_DIR && \
+chmod ug+x $OGG_INSTALL_DIR && \
+yum -y -q install vim-enhanced && \
```

Comment:
So the GoldenGate command line tool, ggsci, uses a Unix editor when setting up configuration for replication data flows.   Each processes configuration is done through this process to create a parameter file.   It could be mapped through an external volume, however ggsci tries to load an external editor (like vi, emacs or nano) when you try to alter the settings in one of these parameter files.    These are usually kept on the same host environment where the source or target database resides.    See: [GGSCI](http://docs.oracle.com/goldengate/c1221/gg-winux/GWUAD/getting-started-oracle-goldengate-process-interfaces.htm#GUID-ED4CCFF1-CE02-4E73-8D9F-790A12B55A7E )for deeper details.